pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -595,6 +590,29 @@ func TestNewCredentialFromLinkedDataProof_EcdsaSecp256k1Signature2019(t *testing
 
 //nolint:lll
 func TestNewCredential_JSONLiteralsNotSupported(t *testing.T) {
+	cmtrJSONLD := `
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "name": "http://schema.org/name",
+    "description": "http://schema.org/description",
+    "image": {
+      "@id": "http://schema.org/image",
+      "@type": "@id"
+    },
+    "hetc": "https://w3c-ccg.github.io/vc-json-schemas/example/cmtr/v0.2/cmtr-v0.2#",
```

Comment:
can we remove these references to w3c-ccg. it doesn't seem needed for the test.

Reply:
right, I will simplify that!

Reply:
Simplified - used `http://127.0.0.1:53401/cmtr.jsonld` dummy context, generated a new valid signature due to `"hetc": "http://localhost:9393/cmtr#"` simplification (no reference to `w3c-ccg`).